src/rules/no_debugger.rs

```diff
@@ -29,6 +29,31 @@ impl LintRule for NoDebugger {
     let mut visitor = NoDebuggerVisitor::new(context);
     visitor.visit_module(module, module);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows the use of the `debugger` statement
+
+`debugger` is a statement which is meant for stopping the javascript execution
+environment and start the debugger at the statement.  Modern debuggers and tooling
+no longer need this statement and leaving it in can cause the execution of your
+code to stop in production.
+    
+### Invalid:
+```typescript
+function isLongString(x:string) {
```

Comment:
```suggestion
function isLongString(x: string) {
```

It would be nice to make the example formatted :)
(if we run `deno fmt --check`, it shows us:

```
1 | -function isLongString(x:string) {
1 | +function isLongString(x: string) {

error: Found 1 not formatted file in 1 file
```